x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/complete/functions/helpers.ts

```diff
@@ -30,6 +31,12 @@ export function getMessageAddedEvents(body: Readable | string) {
   );
 }
 
+export function getConversationCreateEvent(body: Readable | string) {
+  return decodeEvents(body).find(
+    (event): event is ConversationCreateEvent => event.type === 'conversationCreate'
```

Comment:
nit: can you use the enum?

```suggestion
    (event): event is ConversationCreateEvent => event.type === StreamingChatResponseEventType.ConversationCreate
```

Reply:
I've deleted this `getConversationCreateEvent` function, It was duplicated.